files/Trimble_Tekla/Sprint_2_1.ifc

```diff
@@ -0,0 +1,106 @@
+#57= IFCLOCALPLACEMENT(#36,#35);
+#58= IFCALIGNMENT('0yaE5fcMn1Zhp1MvVkuX5K',#9,'Name',$,$,#57,$,$);
+#59= IFCLOCALPLACEMENT(#57,#35);
+#60= IFCCARTESIANPOINTLIST2D(((0.,0.),(30.,0.),(45.,-15.),(60.,0.),(45.,15.)),$);
+#61= IFCINDEXEDPOLYCURVE(#60,(IFCLINEINDEX((1,2)),IFCARCINDEX((2,3,4)),IFCARCINDEX((4,5,2))),.F.);
+#62= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#24,$,.MODEL_VIEW.,$);
+#63= IFCSHAPEREPRESENTATION(#62,'Axis','Curve2D',(#61));
+#64= IFCPRODUCTDEFINITIONSHAPE($,$,(#63));
+#65= IFCALIGNMENTHORIZONTAL('3nMWevyOD289SYM1aXH__V',#9,'Name',$,$,#59,#64);
+#66= IFCCARTESIANPOINTLIST3D(((0.,0.,0.),(30.,0.,-1.),(45.,-15.,-2.),(60.,0.,0.),(45.,15.,2.),(30.,0.,1.)),$);
+#67= IFCINDEXEDPOLYCURVE(#66,(IFCLINEINDEX((1,2)),IFCARCINDEX((2,3,4)),IFCARCINDEX((4,5,6))),.F.);
+#68= IFCSHAPEREPRESENTATION(#62,'Axis','Curve3D',(#67));
+#69= IFCPRODUCTDEFINITIONSHAPE($,$,(#68));
+#70= IFCALIGNMENTVERTICAL('3Sspe6h7X22gs3KYqvgDuK',#9,'Name',$,$,#59,#69);
+#71= IFCCARTESIANPOINT((0.,0.));
+#72= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#71,0.,0.,0.,30.,$,.LINE.);
+#73= IFCALIGNMENTSEGMENT('1sODiIzcvFUvzazSAuaVgW',#9,'Name','Description',$,$,$,#72);
+#74= IFCCARTESIANPOINT((30.,0.));
+#75= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#74,4.71238898038469,15.,15.,47.1238898038469,$,.CIRCULARARC.);
+#76= IFCALIGNMENTSEGMENT('0cVjLJFu9EguI7Oxham0YS',#9,'Name','Description',$,$,$,#75);
+#77= IFCCARTESIANPOINT((60.,0.));
+#78= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#77,1.5707963267949,15.,15.,47.1238898038469,$,.CLOTHOID.);
+#79= IFCALIGNMENTSEGMENT('1yYmkMLjnF8Rjzkzuya2mN',#9,'Name','Description',$,$,$,#78);
+#80= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',0.,30.,0.,-0.0333333333333333,-0.0333333333333333,$,.CONSTANTGRADIENT.);
+#81= IFCALIGNMENTSEGMENT('2CUSA$1aLFKvy$elXBmjJR',#9,'Name','Description',$,$,$,#80);
+#82= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',30.,47.1238898038469,-1.,-0.0424413181578388,0.0848826363156775,-15.0084776206116,.CIRCULARARC.);
+#83= IFCALIGNMENTSEGMENT('23Oh1zFv58qwWyaQeh2cL_',#9,'Name','Description',$,$,$,#82);
+#84= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',77.1238898038469,47.1238898038469,0.,0.0848826363156776,-0.0424413181578387,-15.0084776206116,.PARABOLICARC.);
+#85= IFCALIGNMENTSEGMENT('3TuEcmTe59HB0aJaU2E4zx',#9,'Name','Description',$,$,$,#84);
+#86= IFCREFERENT('3w2rXXoxHDWxnYwiSBdsYw',#9,'Referent name',$,$,#36,$,.BOUNDARY.);
+#87= IFCGROUP('0Iqm6uyOjA3xdYjjWz_1gw',#9,'Referent Group',$,$);
+#88= IFCRELASSIGNSTOGROUP('0y$kFVe5zDuhc32Q5a0jgC',#9,$,$,(#86),$,#87);
+#89= IFCRELAGGREGATES('0b9aNWU25BxhndGrg6uJ_x',#9,$,$,#29,(#37));
+#90= IFCRELAGGREGATES('1wrmqlIlL0Hgp3JSzDsgTC',#9,$,$,#37,(#55,#58));
+#91= IFCRELAGGREGATES('3GDaOSlx92FxhB2EBl8MS_',#9,$,$,#55,(#56));
+#92= IFCRELDEFINESBYPROPERTIES('0ruOpVIB588O3QPluSDd2t',#9,$,$,(#29),#27);
+#93= IFCRELNESTS('3KAF3VtAfAkPySs5h4P2YC',#9,$,$,#58,(#65,#70,#86));

```

Comment:
I think that there should be a separate IfcRelNests for the IfcReferent (not in the same nesting as the horizontal and vertical).